### PR TITLE
[#5370] Windows 64bit package.

### PR DESCRIPTION
**Pull request by dumol**
Scope
=====

Build 64bit Windows package as Windows Nanoserver can't run 32bit binaries.

Changes
=======

Configured Buildbot to also run `python-package` builds on Server 2016.

Build 32bit Windows package on Server 2016.

Build 64bit Windows package on other Windows versions, including our Server 2019 slaves.

Obviously, had to download some 64bit wheels: `cryptography`, `pywin32`, `gmpy2`. The last one was missing at https://pypi.org/project/gmpy2/#files, so I got it from https://www.lfd.uci.edu/~gohlke/pythonlibs/.

Visual Studio 2008 redistributables updated to version `9.0.30729.9518`, using DLLs from Windows Server 2019.

`publicKeyToken` value is no longer removed from Manifest files.

Lots of small related doc improvements.

**Drive-by fixes**:
  * GMP updated from 6.1.2 to 6.2.0
  * PyOpenSSL updated from 19.0.0 to 19.1.0.
  * `pywin32` updated from version 225 to 227.
  * Merged in doc changes from #138.

How to try and test the changes
===============================

reviewers: @adiroiban 

Please review changes.

Run the automated tests.

### CODE REVIEW

**Comment by dumol** [comment]
Had to disable manifest tricks on Windows x64 to get successful builds, not sure if they are still needed.

**needs-review**

File: chevah_build [hunk 1]
```diff
@@ -853,7 +853,7 @@ command_build_python_extra_libraries() {
         execute $PYTHON_BIN -m pip install $PIP_ARGS $library
     done
 
-    if [ $OS == "win" ]; then
+    if [ "$OS" = "win" -a "$ARCH" = "x86" ]; then
```
Comment (adiroiban) [inline chevah_build]: this needs to be reverted as we need to patch on any ARCH and make sure our python has no external dependecies

Comment (dumol) [reply chevah_build]: OK, will return with an updated version that uses own redistributables on X64 as well.

**needs-changes**

**Comment by dumol** [comment]
**needs-changes**

**Comment by dumol** [comment]
Lots of improvements, please revisit PR text.

**needs-review**

**Comment by adiroiban** [comment]
Thanks.

I have create a ticket for chevah/server so that we can run a full test.

https://trac.chevah.com/ticket/5376

Was the package published to staging ?

Mișu, can you please try to create a branch on chevah/server for ticket/5376 and add it as a distributable.

Thanks!

**Comment by dumol** [comment]
> I have create a ticket for chevah/server so that we can run a full test.
> 
> https://trac.chevah.com/ticket/5376
> 
> Was the package published to staging ?

This is done automatically for any successful `python-package` build. And then there's a script that removes packages older than 2 weeks or something.
 
> Mișu, can you please try to create a branch on chevah/server for ticket/5376 and add it as a distributable.

Ack.

**Pull request merged**